superdesk/users/users.py

```diff
@@ -45,22 +44,20 @@ def __init__(self, endpoint_name, app, service, endpoint_schema=None):
                 'readonly': self.readonly
             },
             'display_name': {
-                'type': 'string',
-                'readonly': self.readonly
+                'type': 'string'
             },
             'email': {
                 'unique': True,
                 'type': 'email',
-                'required': True
+                'required': True,
+                'readonly': self.readonly
```

Comment:
Making `phone` field as editable is fine. But as per the video attached in the ticket, I cannot understand why the user does not have the `default` role assigned to the user. Does PA setup have `default` role.